src/common.props

```diff
@@ -14,11 +14,12 @@
   <PropertyGroup>
     <XunitVersion>2.4.1</XunitVersion>
     <XunitRunnerVersion>2.4.3</XunitRunnerVersion>
-    <AkkaVersion>1.4.35</AkkaVersion>
-    <PostgresLowVersion>5.0.11</PostgresLowVersion>
```

Comment:
Actually we might be able to compile with lower version, I could try. However it will not work if you try to use the new PostgreSqlDbBatchQueryExecutor